docker/README.md

```diff
@@ -1,16 +1,22 @@
 # Docker build directory
-
 Put your Dockerfile and other files relating to a docker-build here.
 
 ## Building docker image locally
-
-If you have docker installed on your machine, you may `cd docker; docker build -t my_image:local .`  and build the image.
+If you have docker installed on your machine, you may want to build the image locally:
+```sh
+cd docker
```

Comment:
Already inside the `docker/`
```suggestion
```

Reply:
Not if you read this README from Github 🤷

Reply:
Wait... Yes? If you are reading this README, you have to be sitting inside the docker folder, no matter if you are on github or in your editor. Am I total goof now, what am I misunderstanding? 😅

Reply:
Hehe, no you aren't. I meant that if you read this from Github, you are probably not in `docker` in your terminal - so you would need to cd into it. I just tried to make it foolproof, but nvm this, it's probably not necessary. 

I'll commit your suggestion 😉